docs/serving/creating-domain-mappings.md

```diff
@@ -43,6 +43,9 @@ Service. For example in the example yaml below, the "example.org" Domain
 Mapping maps only "example.org" and not "www.example.org". You can create
 multiple Domain Mappings to map multiple domains and subdomains.
 
+**Note:** If you create a Domain Mapping to map to a [private Knative Service](https://github.com/knative/docs/blob/mkdocs/docs/developer/serving/services/private-services.md), the private Knative Service will be accessible from public internet with the
```

Comment:
Do use `**Note**` now we have mkdocs and there is a new way using [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/)
Use something like
```
!!! Note
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod
    nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor
    massa, nec semper lorem quam in mass.
```

Reply:
Done